judge/fixtures/language_small.json

```diff
@@ -263,7 +263,7 @@
          "name":"Brain****",
          "short_name":null,
          "common_name":"Brain****",
-         "ace":"bf",
+         "ace":"plaintext",
```

Comment:
```suggestion
         "ace":"plain_text",
```

Looks like the name should actually be `plain_text`.

Reply:
Yep that looks more correct, thanks for pointing it out 😅